agents/agents-core/src/commonMain/kotlin/ai/koog/agents/core/dsl/builder/AIAgentGraphStrategyBuilder.kt

```diff
@@ -9,7 +9,7 @@ import kotlin.reflect.typeOf
 
 /**
  * A builder class responsible for constructing an instance of `AIAgentStrategy`.
```

Comment:
nit: Let's from now on replace backticks with square brackets when refering to types? It will create a proper reference in generated API docs, making navigation slightly easier